.github/scripts/get_release_version.py

```diff
@@ -20,14 +20,25 @@
 
 gitRef = os.getenv("GITHUB_REF")
 tagRefPrefix = "refs/tags/v"
+nightlyTagPrefix = "refs/tags/nightly"
 
 with open(os.getenv("GITHUB_ENV"), "a") as githubEnv:
 
-    if gitRef is None or not gitRef.startswith(tagRefPrefix):
+    if gitRef is None or not gitRef.startswith((tagRefPrefix, nightlyTagPrefix)):
         githubEnv.write("REL_VERSION=edge\n")
+        githubEnv.write("RELEASE_TO_GH=False\n")
         print ("This is daily build from {}...".format(gitRef))
         sys.exit(0)
 
+    githubEnv.write("RELEASE_TO_GH=True\n")
+
+    if gitRef.find("nightly") > 0:
```

Comment:
We cannot have a full release for nightly build because of 2 reasons:
1. Helm chart needs to follow semantic versioning, nightly-X-Y-Z format is not a valid format for that.
2. Even if there is a workaround for (1), it would "explore" the number of helm chart entries in the index file: https://github.com/dapr/helm-charts/blob/master/index.yaml


My proposal is to only release the dapr images to Docker Registry and not release Helm charts as part of the nightly release. People that need to use any of the nightly images can do so by using the latest release Helm + changing the image of specific containers to use the nightly build.

Reply:
@artursouza I see that we already push an edge image to Docker after new changes are merged into master. I was wondering how would this nightly build docker image differ from that? If people want the most recent changes can they not use the edge image that is published?

Reply:
> @artursouza I see that we already push an edge image to Docker after new changes are merged into master. I was wondering how would this nightly build docker image differ from that? If people want the most recent changes can they not use the edge image that is published?

Yes, there is a big difference. The nightly build is a fixed point in time while the `edge` tag is a moving target. You can depend on a nightly build with guarantees it will never change. The `edge` tag is always changing. We, for example, plan to use nightly build to allow SDK builds to use a runtime version that is not yet released but not a moving target to keep the build reproducible. I hope that clarifies it.

Reply:
Yes that makes sense, thank you. I will make the changes